user_print.py

```diff
@@ -0,0 +1,22 @@
+
+
+def farewell_greeting_to_user():
+    print("Thank you for choosing to look at some pokemon with us! have a pleasant day")
+
+def ask_user_draw_pokemon():
+    print("Would you like to draw a pokemon card?")
```

Comment:
That's what I meant in my previous comment - You're not `ask`ing, you're, at most, `tell`ing the user what you're going to ask, or even more accurate - you're `tell`ing the user what question they're supposed to be answering soon. \
Simply use `input('Would you like to draw a pokemon card?')` and return the answer